packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -25,6 +25,17 @@ const int _kMaxDroppedSwipePageForwardAnimationTime = 800; // Milliseconds.
 // user releases a page mid swipe.
 const int _kMaxPageBackAnimationTime = 300; // Milliseconds.
 
+/// Barrier color used for a barrier visible during transitions for Cupertino
+/// page routes. This barrier color is only used for full-screen page routes with
```

Comment:
A new line in between sentences.

Reply:
Gotcha. Should that always be the case? I see some examples of when documentation is written in paragraph form—I imagine it should be split up in this case just because the sentences are addressing two different things (first, what the field is, and then second, how it's used)?

Reply:
Yes we followed the dart guideline https://dart.dev/guides/language/effective-dart/documentation#doc-comments

`A single-sentence summary of what the library is for.`

If you spot a doc string that has multiple sentences in the first paragraph, it is likely a mistake

Reply:
Cool cool thanks for explaining! Done